presto-main/src/main/resources/webapp/index.html

```diff
@@ -250,8 +265,38 @@
     return [].concat.apply([], array);
 }
 
-setInterval(function () { redraw(); }, 1000);
+var flipRefresh = (function() {
```

Comment:
This seems very complicated. Why not keep the timer always enabled and have a global `paused` flag?